javascript/javascript2/week1/ShortestWord.js

```diff
@@ -0,0 +1,25 @@
+
+const danishWords= ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
+console.log(getShortestWord(danishWords));
+
+
+function getShortestWord() {
+  if(danishWords == null || danishWords.length < 1){
```

Comment:
It is a very good idea to check the array first.
It is another way that might be interesting for you.

    if (!Array.isArray(danishWords)) {
     console.log("it's not an array");
    }
    //If it is an array, check its length property
    if (danishWords.length == 0) {
    console.log("the array is empty");
    }